### PR TITLE
Geography Groupings

### LINKED ISSUE #3: geography_groupings

### PR DESCRIPTION
**Pull request by tomgiddingsjisc**

### CODE REVIEW

File: tables/geography_groupings.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+|N92000002|Northern Ireland|26|
+|S92000003|Scotland|32|
+|W92000004|Wales|22|
+
+
+## Schema
+
+|column|type|use|
+|-|-|-|
+|id|int4|Primary key.|
```
Comment (davidplumb) [inline tables/geography_groupings.md]: I think this needs more explanation, it is simply an id but it has much more context in the database than a lot of the other ids

File: tables/geography_groupings.md [hunk 2]
```diff
@@ -1 +1,82 @@
...
+|abbreviation|varchar(100)|Short code/abbreviation for a grouping, such as `CNTY` for `County`.|
+|name|varchar(100)|An end-user friendly name for a grouping, such as `Wards and Electoral Divisions`.|
+|description|text|An end-user friendly description for a grouping.|
+|top_level_geography_id|int4|The top-level geography for a given grouping, for example `Regions` has a top-level geography of `England` as the concept of a region exists in England only.|
```
Comment (davidplumb) [inline tables/geography_groupings.md:55]: description of top_level_geography is good, I think that the `id` column in the schema needs as much description as this as they are of equal importance for the user to understand what this table is about

**Review by davidplumb — COMMENTED** [review]
I think one other table you need to reference in this document is `top_level_geographies`
it's important to point out that by combining the ID of this table with the ID of `top_level_geographies` you get a grouping that can be mapped to a specific area

e.g. 
a geography_groupings id of 2000 combined with a top_level_geographies id of 1 represents the entirety of the United Kingdom (2000:1). 

In the previous dataset we referred to the first ID as "geolocations", and the top level ID as "extents" this terminology might be outdated now though but it's really important that we can refer to both these IDs by a common name and I think we should detail that in this document, or should reference another readme that gives an overview of the naming convention

File: tables/geography_groupings.md (hunk 2 above)
Comment (tomgiddingsjisc) [reply tables/geography_groupings.md:55]: Sure thing, will add more detail here.

**Comment by tomgiddingsjisc** [comment]
> I think one other table you need to reference in this document is `top_level_geographies` it's important to point out that by combining the ID of this table with the ID of `top_level_geographies` you get a grouping that can be mapped to a specific area

Yep, that's a good shout. I've added this to both the 'What are geography groupings?' section and the schema.

> In the previous dataset we referred to the first ID as "geolocations", and the top level ID as "extents" this terminology might be outdated now though but it's really important that we can refer to both these IDs by a common name and I think we should detail that in this document, or should reference another readme that gives an overview of the naming convention

I think we should definitely make use of a terminology/glossary page. In terms of namings, we're probably best sticking as close to the column names as possible to avoid introducing duplicate terms or adding confusion!

File: tables/geography_groupings.md (hunk 2 above)
Comment (tomgiddingsjisc) [reply tables/geography_groupings.md:55]: Detail added to `id`

File: tables/geography_groupings.md (hunk 1 above)
Comment (tomgiddingsjisc) [reply tables/geography_groupings.md]: Added detail here

**Review by davidplumb — APPROVED** [review]
looks much better, approved

**Pull request merged**